src/pages/SupportPage.ts

```diff
@@ -0,0 +1,38 @@
+// import _ from 'lodash/fp';
+import { h, h2, p } from 'react-hyperscript-helpers';
+import FooterWrapper from 'src/components/FooterWrapper';
+import { PageBox } from 'src/components/PageBox';
+import TopBar from 'src/components/TopBar';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+import { SupportResourceList } from 'src/support/SupportResourceList';
+
+interface SupportPageProps {
+  queryParams: {
+    selectedType: string | undefined;
+    resourceName: string | undefined;
+  };
+}
+
+const SupportPage = (props: SupportPageProps) => {
+  const selectedType = props.queryParams.selectedType;
+  const resourceName = props.queryParams.resourceName;
+
+  return h(FooterWrapper, [
+    h(TopBar, { title: 'Support', href: Nav.getLink('support') }, []),
+    h(PageBox, { role: 'main' }, [
+      h2({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Support']),
+      p(['Select resource type.']),
+      h(SupportResourceList, { queryParams: { selectedType, resourceName } }),
+    ]),
+  ]);
+};
+
+export const navPaths = [
+  {
+    name: 'support',
+    path: '/support',
```

Comment:
I would recommend something like "support-internal" because anyone will be able to hit this page if the try the URL. It think it would be good if it makes it a bit clearer that this isn't a page for general support purposes.